docs/getting-started/setup.mdx

```diff
@@ -85,6 +84,34 @@ the database engine, you'll need to run your Docker container slightly different
       /app/run_app.sh mage start another_repo
    ```
 
+### Using MSSQL as database
+
+Besides SQLite and PostgreSQL, MSSQL can also be used as the database engine, as shown in the following steps using Docker container:
+
+1. Create Docker network
+
+   ```bash
+   docker network create mage-app
+   ```
+
+2. Start MSSQL Docker container in a separate window
+
+   ```bash
+   docker run --network mage-app --network-alias mssql_db \
+   -it -p 1433:1433 -e "ACCEPT_EULA=Y" \
+   -e "MSSQL_SA_PASSWORD=test_Username123" \
+   mcr.microsoft.com/mssql/server:2022-latest
+   ```
+
+3. Launch Mage with `MAGE_DATABASE_CONNECTION_URL` environment variable
+
+   ```bash
+   docker run --network mage-app -it -p 6789:6789 -v $(pwd):/home/src \
+   -e MAGE_DATABASE_CONNECTION_URL="mssql+pyodbc://?odbc_connect=DRIVER={ODBC Driver 18 for SQL Server};SERVER=mssql_db;DATABASE=master;UID=SA;PWD=test_Username123;ENCRYPT=yes;TrustServerCertificate=yes;" \
+   mageai/mageai \
+   /app/run_app.sh mage start another_repo_mssql
```

Comment:
let's update the command with different parameters, for example `/app/run_app.sh mage start <project_name>` or something. can you help change it in the postgres command as well?

Reply:
Good catch: made the suggested change.

Reply:
can you also update the rest of the command? for example, in the `MAGE_DATABASE_CONNECTION_URL` field, it's not clear to me which fields the user might need to change.

Reply:
Sure, added the explanation for these parameters.